pom.xml

```diff
@@ -225,13 +234,21 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-gpg-plugin</artifactId>
+                        <version>1.6</version>
                         <executions>
                             <execution>
                                 <id>sign-artifacts</id>
                                 <phase>verify</phase>
                                 <goals>
                                     <goal>sign</goal>
                                 </goals>
+                                <configuration>
+                                <!-- This is necessary for gpg to not try to use the pinentry programs -->
```

Comment:
It's a collection of programs :)
https://gnupg.org/software/pinentry/index.html